lib/srv/exec.go

```diff
@@ -60,6 +60,8 @@ type ExecResult struct {
 
 // Exec executes an "exec" request.
 type Exec interface {
+	fmt.Stringer
```

Comment:
no need to add it in the interface
as long as `remoteExec` implements `fmt.Stringer`, printing it with `%v` should pick it up